<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenIndicatorsWidget</class>
 <widget class="QWidget" name="GenIndicatorsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: transparent;</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>361</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QProgressBar" name="signalLevel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk {
	background-color: white;
	margin: 3px;
}

QProgressBar {
background: transparent;
color: white;
border: none;
margin: 3px;
text-align: center;
}
</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="signalIcon">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources/cutebox.qrc">:/images/antenna_icon.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="clock">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
color: white;</string>
      </property>
      <property name="text">
       <string>23:45</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="batteryIcon">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources/cutebox.qrc">:/images/battery_icon.png</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="batteryLevel">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar::chunk {
	background-color: white;
	margin: 3px;
}

QProgressBar {
background: transparent;
color: white;
border: none;
margin: 3px;
text-align: center;
}</string>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/cutebox.qrc"/>
 </resources>
 <connections/>
</ui>
